webapp/templates/jury/partials/external_contest_info.html.twig

```diff
@@ -43,6 +43,8 @@
                 <td>
                     {% if not externalContestSource.lastPollTime %}
                         Event feed reader never checked in.
+                    {% elseif externalContestSource.lastHTTPCode is not same as(200) %}
+                        Received non HTTP200 ({{ externalContestSource.lastHTTPCode }}).
```

Comment:
```suggestion
                        Received non HTTP 200 ({{ externalContestSource.lastHTTPCode }}).
```

I don't think `HTTP200` is a single word. Or maybe even `non-200 HTTP code` or `Received {{ externalContestSource.lastHTTPCode }} HTTP code`?